QA.md

```diff
@@ -201,19 +201,45 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
-    1. Confirm the Timelock checkbox is also disabled.
-    1. Click on 'Add another token' button. A modal is displayed saying the action is not supported.
+    1. Confirm that the token dropdown list can be clicked and show all tokens.
+    1. Confirm the Timelock checkbox is disabled.
     1. Try sending a transaction to an invalid address. An error message should show up on the wallet. Nothing should be displayed on Ledger.
     1. Try sending a transaction with an amount bigger than the one you have. An error message should show up on the wallet. Nothing should be displayed on Ledger.
-    1. Send tokens to both copied addresses in the same transaction (2 outputs). A modal will be displayed asking to confirm operation on Ledger.
+    1. Send HTR tokens to both copied addresses in the same transaction (2 outputs). A modal will be displayed asking to confirm operation on Ledger.
     1. On Ledger, it should show 'Output 1/2' on the first step the address on the second and the amount on the third step (steps being from left to right), then an "Approve" and "Reject" steps. After clicking both buttons on the "Approve", it shows 'Output 2/2' and the other output (address and ammount).
     1. Clicking both buttons on the "Approve" step one more time will display the confirmation screen. Reject signing the transaction. The modal will hide and an error message should appear on the wallet.
     1. Click to send the transaction again. This time, approve it on the signing screen. Ledger screen will show 'Processing' for a while and the desktop wallet will display the modal while solving proof of work.
     1. After transaction is completed, it closes the modal and goes back to main wallet screen. The first transaction on the list must have value 0.00.
+    1. Go to the custom token wallet page and confirm that the key icon is showing next to the name (and not a checkmark).
+    1. If there is a checkmark icon, go to settings and untrust all tokens.
+    1. Click on 'Send tokens' on the top menu again.
+    1. Change the token dropdown to the custom token and try to send any amount. A modal will be displayed saying there are unverified custom tokens and the list should have the custom token added to the transaction.
+    1. Go to the custom token wallet screen and sign the custom token with Ledger (key icon next to the name).
+    1. Go back to the 'Send tokens' screen and click on 'Add another token' button and send HTR to an address and the custom token to the second copied address. Make sure both amounts are less than the balance available. A modal will be displayed asking to confirm operation on Ledger.
+    1. On Ledger you should check that each token is being sent to the correct address then click both buttons on the 'Approve' screen.
+    1. After transaction is completed, it closes the modal and goes back to main wallet screen. The first transaction on the list must have value 0.00.
```

Comment:
The value will be shown as `0.00` only if the two output addresses selected on step 1 are from the same wallet.
If the addresses are for another wallet, the full value will be shown.

Maybe it's a good idea to specify on step 1 that the output addresses to be used here must be from the same wallet as the input.

Reply:
Done in [e58aa9e](https://github.com/HathorNetwork/hathor-wallet/pull/278/commits/e58aa9ebd61ab0f3106e83000d7c1b28a36638ea)